demos/demo-donut.js

```diff
@@ -148,18 +148,31 @@ function createDonutWithHighlightSliceChart() {
     }
 }
 
+function createLoadingState() {
+    let donutChart = donut(),
+        donutContainer = d3Selection.select('.js-loading-container'),
+        containerWidth = donutContainer.node() ? donutContainer.node().getBoundingClientRect().width : false,
+        dataset = null;
+
+    if (containerWidth) {
+        donutContainer.html(donutChart.loadingState());
```

Comment:
I was expecting something like:
```
  donutChart.width(containerWidth);

  donutContainer.datum(dataset).call(donutChart);
```

Would this work?

Reply:
This wouldn't work right now unfortunately. Right now we're just providing the loading states, but the charts themselves don't do anything with the states. This was just the first step (making the states available for clients like britecharts-react).

Reply:
awesome!